hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsService.java

```diff
@@ -64,11 +64,14 @@
     void addAlertTags(String tenantId, Collection<String> alertIds, Map<String, String> tags) throws Exception;
 
     /**
-     * Persist the provided events and send to the engine for alerts evaluation.
+     * Send events to the engind for alerts evaluation.
+     * Persist the provided events if persist flag is set to true.
+     *
      * @param events Set of unpersisted Events.
+     * @param persist if true events are persisted
      * @throws Exception any problem
      */
-    void addEvents(Collection<Event> events) throws Exception;
+    void addEvents(Collection<Event> events, boolean persist) throws Exception;
```

Comment:
The events/data endpoint is a good addition to the API.   It may be nice thought if addEvents() remained unchanged and perhaps we add a new signature, sendEvents(), that is analogous to the already-existing sendData().